Menu/GameShell/20_Retro Games/MGBA/action.config

```diff
@@ -4,3 +4,4 @@ EXT=gb,gbc,gba,gbx
 LAUNCHER=retroarch -L
 TITLE=MGBA Roms
 SO_URL=https://raw.githubusercontent.com/cuu/emulators/master/mgba_libretro.so.zip
+RETRO_CONFIG=/home/cpi/apps/launcher/Menu/GameShell/20_Retro\ Games/MGBA/retroarch-local.cfg
```

Comment:
Why are we calling this retroarch-local.cfg and not something like MGBA-remap.cfg ?

Reply:
so what about MAME NESTOPIA and others later?
that's just a little complex on naming

Reply:
Right but wouldn't it be easier yo keep track of if each one had a unique name? Like naming variables it seems like it woulf be a good idea. Doesnt have to be the whole name. Maybe Im wrong.

Reply:
you are right, easier to keep track
but not easier for coding...
That's just a bit of confusing right now for me
lol 
:)

Reply:
btw
launcher itself now won't track any .cfg file